aten/CMakeLists.txt

```diff
@@ -379,6 +380,9 @@ add_subdirectory(src/THS)
 if(NO_CUDA)
   message("disabling CUDA because NO_CUDA is set")
 else()
+  ADD_DEFINITIONS(-DAT_CUDA_ENABLED)
+  INCLUDE_DIRECTORIES(${CUDA_INCLUDE_DIRS})
+  SET(CUDA_FLAG -n)
```

Comment:
This is a build fix! Previously AT_CUDA_ENABLED was never passed to the test files (because it was set in a sub-CMakeFile), now it is. (I fixed this differently in #3666)